tracee-ebpf/tracee/user_mode_events.go

```diff
@@ -36,27 +36,33 @@ func CreateInitNamespacesEvent() (external.Event, error) {
 }
 
 // getInitNamespaceArguments Fetch the namespaces of the init process and parse them into event arguments.
-func getInitNamespaceArguments() []external.Argument {
-	initNamespaces := fetchInitNamespaces()
+func getInitNamespaceArguments() ([]external.Argument, error) {
+	initNamespaces, err := fetchInitNamespaces()
+	if err != nil {
+		return nil, err
+	}
 	initNamespacesArgs := make([]external.Argument, len(EventsIDToParams[InitNamespacesEventID]))
 	for i, arg := range initNamespacesArgs {
 		arg.ArgMeta = EventsIDToParams[InitNamespacesEventID][i]
 		arg.Value = initNamespaces[arg.Name]
 		initNamespacesArgs[i] = arg
 	}
-	return initNamespacesArgs
+	return initNamespacesArgs, nil
 }
 
 // fetchInitNamespaces fetch the namespaces values from the /proc/1/ns directory
-func fetchInitNamespaces() map[string]uint32 {
+func fetchInitNamespaces() (map[string]uint32, error) {
 	initNamespacesMap := make(map[string]uint32)
 	namespaceValueReg := regexp.MustCompile(":[[[:digit:]]*]")
-	namespacesLinks, _ := ioutil.ReadDir(InitProcNsDir)
+	namespacesLinks, err := ioutil.ReadDir(InitProcNsDir)
```

Comment:
In case of permission errors and running this code along with other unit tests we don't see why it's failing

Reply:
I suggest to skip the error, and create a GitHub issue for it, in general this test should be part of integration test. @simar7 FYI

Reply:
Skipped and created #1228